test/e2e-api/admin/settings.test.js

```diff
@@ -207,6 +207,38 @@ describe('Settings API', function () {
                     });
                 });
         });
+
+        it('Can attempt to connect to stripe', async function () {
```

Comment:
We can maybe improve this test by first doing a request to connect to stripe, I'm not sure how tricky it will be but I can take a look if desired

Reply:
It would be improved by not have an error response, and actually setting fake stripe keys in the DB